.github/workflows/magma-build-openvswitch.yml

```diff
@@ -0,0 +1,110 @@
+# Copyright 2022 The Magma Authors.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Magma Build & Publish Open vSwitch Artifacts
+on:
+  push:
+    branches:
+      - master
+      - 'v1.*'
+    paths:
+      - third_party/gtp_ovs/ovs-gtp-patches/**
+      - .github/workflows/magma-build-openvswitch.yml
+  pull_request:
+    types: [ opened, reopened, synchronize ]
+    branches:
+      - master
+      - 'v1.*'
+    paths:
+      - third_party/gtp_ovs/ovs-gtp-patches/**
+      - .github/workflows/magma-build-openvswitch.yml
+  schedule:
+    - cron: 36 4 * * 0
+  workflow_dispatch:
+    inputs:
+      production:
+        description: Is production build?
+        type: boolean
+        default: false
+      repository:
+        description: Repository to push to?
+        type: choice
+        default: magma-packages-test
+        options: [ magma-packages-test, magma-packages-prod ]
```

Comment:
Why do we even have the option `magma-packages-prod`? The release workflow is to copy the current focal-ci state to `magma-packages-prod/focal-1.8.0` - i.e., we usually do not publish directly to `magma-packages-prod`.

Reply:
I will change it. However, I take that from the `magma-build-3rd-party` workflow. Do we have the same problem there?

Reply:
Maybe wait with changing this - let me try to clarify if there is a plan to change this workflow with the lf artifactory.

Reply:
@nstng, should we pin the `repository` to 'magma-packages-test', meaning no custom input allowed?